manifests/subnet.pp

```diff
@@ -0,0 +1,69 @@
+# Define a DHCP subnet
+# @param network
+#   The network range (without the mask) from where IP's will be served
+# @param mask
+#   The subnet mask for our network 
+# @param pools
+#   Specify a pool of addresses that will be treated differently than another pool of addresses,
+#   even on the same network segment or subnet.
+# @param gateway
+#   This option specifies a list of comma-separated IP addresses for routers on the client's subnet.
+#   Routers should be listed in order of preference.
+# @param options
+#   Custom DHCP option statements
+#   DHCP option statements always start with the option keyword, followed by an option name, followed by option
+#   data. The option names and data formats are described below. It is not necessary to exhaustively specify all
+#   DHCP options - only those options which are needed by clients
+# @param parameters
+#   Custom DHCP parameters
+#   Each element is added as a separate line
+# @param mtu
+#   This option specifies the MTU to use on this interface. The minimum legal value for the MTU is 68. 
```

Comment:
If the minimum legal value is 68, then I wonder if the data type should reflect this. Probably better for a separate PR to preserve compatibility.